src/containers/addNew.js

```diff
@@ -0,0 +1,193 @@
+import React, { Component } from 'react';
+import Ratings from 'react-ratings-declarative';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import './form.scss';
+import { addNewBeer } from '../actions';
+
+class AddNew extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      touched: {
+        beerName: false,
+        rating: false,
+        desc: false,
+      },
+      beerName: '',
+      rating: 1,
+      desc: '',
+      file: '',
+      isDisabled: true,
+    };
+  }
+
+  changeTheRating = newRating => {
+    this.setState({ rating: newRating });
+  };
+
+  handleChange = field => event => {
+    this.setState(
+      {
+        [field]: event.target.value,
+      },
+      () => {
+        if (this.state.beerName !== '' && this.state.desc !== '') {
+          this.setState({
+            isDisabled: false,
+          });
+        } else
+          this.setState({
+            isDisabled: true,
+          });
+      },
+    );
+  };
```

Comment:
Lines 38-45 should be some other function called here (checkDisabledFields or soemthing like that)